src/jobskills/discord/jobs/worker.py

```diff
@@ -47,15 +40,23 @@ async def scrape_handler(arq_ctx, url: str):
 
 async def message_edit(arq_ctx, dc_ctx, msg: str):
     try:
-        dc_ctx.edit(Message(content=msg))
+        if len(msg) < settings.discord.msg_max_len:
```

Comment:
seems to be a required setting, no validator?

Reply:
Now present in the main settings Python file (to avoid also running validators for Discord settings that aren't present in the workers).